mirdata/salami.py

```diff
@@ -109,7 +109,7 @@ def load_track(track_id, data_home=None):
             'annotator_1_time': None, 'annotator_2_time': None, 'class': None,
             'genre': None
         }
-    salami_path = get_local_path(data_home, SALAMI_DIR)
+    salami_path = utils.get_local_path(data_home, SALAMI_DIR)
```

Comment:
Is there a `from utils import get_local_path` line that we should remove?

Reply:
Nope - it was a mistake, not an import style error!